src/forms/forms.stories.html

```diff
@@ -0,0 +1,134 @@
+            <div class="omrs-radio-button">
+              <input
+                type="radio"
+                disabled
+                id="radio1"
+                checked="true"
+                name="radio1"
+                group="radio"
+              />
+              <label for="radio1">Active Radio</label>
+            </div>
+            <div class="omrs-radio-button">
+              <input
+                type="radio"
+                disabled
+                id="radio2"
+                name="radio1"
+                group="radio"
+              />
+              <label for="radio2">Inactive Radio</label>
+            </div>
+          </div>
+        </form>
+      </div>
+    </div>
+
+    <h4>Code example:</h4>
+    <p class="type-demo omrs-bold">
+      <%=determineInputType(inputType)%> input
+    </p>
+    <div class="omrs-input-group">
+      <div class="<%=determinStyle(inputType)%>">
+        <input type="<%=determineInputType(inputType)%>" "<%=isChecked%>"
+        id="inputHtml" />
+        <label htmlFor="inputHtml">
+          <%=inputType%>
+        </label>
+      </div>
+    </div>
+    <div class="examples">
+      <div class="tabs">
+        <div lang="html" class="tab">HTML</div>
+        <div lang="jsx" class="tab">JSX</div>
+      </div>
+      <div class="panels">
+        <div class="panel html">
+          <div class="stories-example lang-html">
+            <div className="omrs-checkbox">
+              <input type="checkbox" id="inputHtmlCheckBox" />
+              <label for="inputHtmlCheckBox">CheckBox HTML</label>
+            </div>
+          </div>
+          <div class="stories-example lang-html">
+            <div className="omrs-radio-button">
+              <input type="radio" id="inputHtmlRadio" />
+              <label for="inputHtmlRadio">Radio HTML</label>
+            </div>
+          </div>
+        </div>
+        <div class="panel jsx">
+          <div class="stories-example lang-html">
+            <div className="omrs-checkbox">
+              <input type="checkbox" id="inputJsxCheckBox" />
+              <label htmlFor="inputJsxCheckBox">CheckBox JSX</label>
+            </div>
+          </div>
+          <div class="stories-example lang-html">
+            <div className="omrs-radio-button">
+              <input type="radio" id="inputJsxRadio" />
+              <label htmlFor="inputJsxRadio">Radio JSX</label>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</article>
+<style>
+  div.container {
+    display: flex;
+    justify-content: flex-start;
+    margin-top: 30px;
+    border: 1px solid whitesmoke;
+    border-radius: 4px;
+    width: 100%;
+    padding-left: 20px;
+  }
+
+  h3.title {
+    text-align: center;
+    margin-bottom: 45px;
+    color: #1ea7fd;
+  }
+
+  .stories-example > input {
+    margin: 35px;
+  }
+
+  p.type-demo {
+    text-transform: capitalize;
+  }
+
+  .omrs-input-group {
```

Comment:
Since this css class isn't exported from the styleguide, I don't think it should be named with the `omrs-` prefix. People looking at the HTML for inputs might be confused when they see this. It might be best to do a selector like this so that the copy/pasteable example code doesn't have css classes that aren't in the styleguide:

```css
.form-examples form > div {
}
```

```html
<article class="form-examples">
</article>
```

^ this avoids the code in the examples not referencing css classes that aren't actually part of the styleguide.